frontend/js/modules/group-watch.js

```diff
@@ -0,0 +1,79 @@
+/**
+*  # Group watching module
+*
+*  ## License
+*
+*  Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Please remove the license for newly created files (will determine licensing later on)